### DISCUSSION TITLE
Firmware Phrasing

### DISCUSSION THREAD

**Discussion by lloydpick** [phnxdrgn536/FLSun-v400#21] [General]
I'd suggest a somewhat rephrasing of the firmware versions away from "stock" and "reflashed" to perhaps...

"Klipper Version: FLSun or Official"

Since stock could easily be misinterpreted while also someone could be confused by "reflashed" meaning that they have re-installed the FLSun image, rather than switching to the Guilouz (official klipper) version which has the standardized macro naming.

**Comment by phnxdrgn536** [comment]
I’ll take a poll and see what people think about it.. but really the only
difference is the pre loaded start and end G-code lol the “stock” profile
will work for either but the reflashed uses macro codes that are not on the
original FLSun pad
